torch/csrc/jit/export.cpp

```diff
@@ -1331,6 +1348,9 @@ std::tuple<std::string, RawDataExportMap> export_onnx(
     ::torch::onnx::OperatorExportTypes operator_export_type,
     bool strip_doc_string,
     bool keep_initializers_as_inputs) {
+  auto old_logger_handler =
+      ::google::protobuf::SetLogHandler(&protobuf_log_handler);
```

Comment:
Should we make this to a global, when the protobuf logger is initialize, If yes, where should it live?